packages/odie-client/src/components/message/messages-container.tsx

```diff
@@ -29,10 +38,20 @@ export const MessagesContainer = forwardRef< HTMLDivElement, ChatMessagesProps >
 			if ( message.type === 'error' ) {
 				lastErrorMessageIndex = index;
 			}
+			// Find the first business message
+			// this will be used to notify users that they are talking with a human.
+			if ( firstBusinessMessageIndex === -1 && message.role === 'business' ) {
+				firstBusinessMessageIndex = index + 1;
+			}
```

Comment:
Is this necessary? Based on the Figma it looks like this is only showing when the transfer to Zendesk happens. So when the user pulls up the conversation again later it shouldn't be there. It may be better to add a flag to the transfer message [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/odie-client/src/query/use-create-zendesk-conversation.ts#L22-L24) like `transfer-to-support` then use that flag to display the "Chatting with support now" message. That way when the conversation is loaded from the history that piece isn't there.

Also, this seems to be putting the component BEFORE the first user message but only after I send a response. I think it should be directly after the final bot response.

<img width="413" alt="Screen Shot 2024-10-29 at 11 24 29" src="https://github.com/user-attachments/assets/9dc406a7-e15f-462f-97e3-69c27bf2c16e">